nixos/modules/services/audio/navidrome.nix

```
@@ -0,0 +1,68 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.navidrome;
+  settingsFormat = pkgs.formats.json {};
+in {
+  options = {
+    services.navidrome = {
+      enable = mkEnableOption pkgs.navidrome.meta.description;
+      settings = mkOption rec {
```

some white space to help differentiate the different options
```suggestion
    services.navidrome = {

      enable = mkEnableOption pkgs.navidrome.meta.description;
 
      settings = mkOption rec {
```